XYZ Company has offices in four different zones. The company wishes to
investigate the following :
1. The mean sales generated by each zone.
2.Total sales generated by all the zones for each month.
3.Check whether all the zones generate the same amount of sales.
Help the company to carry out their study with the help of data provided

In [1]:
import pandas as pd
import numpy as np

# Reading the dataset into python

In [2]:
data=pd.read_csv(r'C:\Users\Abhilash\Downloads\Sales_data_zone_wise.csv')

In [3]:
data.head()

Month  Zone - A  Zone - B  Zone - C  Zone - D
0  Month - 1   1483525   1748451   1523308   2267260
1  Month - 2   1238428   1707421   2212113   1994341
2  Month - 3   1860771   2091194   1282374   1241600
3  Month - 4   1871571   1759617   2290580   2252681
4  Month - 5   1244922   1606010   1818334   1326062

In [4]:
data.isna().sum()

Month       0
Zone - A    0
Zone - B    0
Zone - C    0
Zone - D    0
dtype: int64

There is no null value in the given data

In [5]:
data.describe()

Zone - A      Zone - B      Zone - C      Zone - D
count  2.900000e+01  2.900000e+01  2.900000e+01  2.900000e+01
mean   1.540493e+06  1.755560e+06  1.772871e+06  1.842927e+06
std    2.619401e+05  1.683899e+05  3.331937e+05  3.750165e+05
min    1.128185e+06  1.527574e+06  1.237722e+06  1.234311e+06
25%    1.305972e+06  1.606010e+06  1.523308e+06  1.520406e+06
50%    1.534390e+06  1.740365e+06  1.767047e+06  1.854412e+06
75%    1.820196e+06  1.875658e+06  2.098463e+06  2.180416e+06
max    2.004480e+06  2.091194e+06  2.290580e+06  2.364132e+06

# 1.The mean sales generated by each zone

In [6]:
data_mean=data[['Zone - A','Zone - B','Zone - C','Zone - D']].mean()
print(data_mean)

Zone - A    1.540493e+06
Zone - B    1.755560e+06
Zone - C    1.772871e+06
Zone - D    1.842927e+06
dtype: float64


Among the 4 zones the mean sale is highest in zone-D and lowest for zone-A.

# 2.Total sales generated by all the zones for each month

In [7]:
data_sum=data.sum(axis=1)
print(data_sum)

0     7022544
1     7152303
2     6475939
3     8174449
4     5995328
5     7151387
6     7287108
7     7816299
8     6703395
9     7128210
10    7032783
11    6111084
12    5925424
13    7155515
14    5934156
15    6506659
16    7149383
17    7083490
18    6971953
19    7124599
20    7389597
21    7560001
22    6687919
23    7784747
24    6095918
25    6512360
26    6267918
27    7470920
28    6772277
dtype: int64


By calculating the total sales of all zones in each month the highest sales occur in month 3-(817449) and the lowest sale occur in month 12-(5925424)

# 3.Checking whether all the zones generate the same amount of sales

### H0:-amount of sales in all zone are equal

### H1:-amount of sales in each zone differ

In [8]:
from scipy.stats import f_oneway
f_oneway(data['Zone - A'],data['Zone - B'],data['Zone - C'],data['Zone - D'])

F_onewayResult(statistic=5.672056106843581, pvalue=0.0011827601694503335)

by applying anova test we get f value as 5.672 and p value 0.001 that is nearly equal to 0.so that we reject the null hypothesis and accept alternative hypothesis that is amount of sales in each zone is different